### DISCUSSION TITLE
Installing user scoped applications while running in a SYSTEM context

### DISCUSSION THREAD

**Discussion by Ankitr19** [microsoft/winget-cli#4412] [General]
I am trying out installing applications via winget on devices, and in this case winget is running in the SYSTEM context. For some of the apps like `Postman` which only supports user scope installation doesn't work. Is there a way that this can be done? For example by explicitly invoking the winget executable present in the user's directory

**Comment by Trenly** [comment]
Because winget installs user scoped applications using the current context, there currently is not a way to install user scoped applications as SYSTEM to a specific user account
